stories/Button.stories.ts

```diff
@@ -53,6 +54,37 @@ export const OnlyIcon: Story = {
   },
 };
 
+export const Link: Story = {
+  args: {
```

Comment:
What do you think of adding `openInNewTab ` to the argTypes in order be selectable in storybook?